packages/google-cloud-run/src/index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
+const { util: coreUtil } = require('@instana/core');
 const { environment: environmentUtil } = require('@instana/serverless');
 
-environmentUtil.validate();
+let isExcludedFromInstrumentation = coreUtil.excludedFromInstrumentation && coreUtil.excludedFromInstrumentation();
```

Comment:
Same as before, `const` should be enough :)

Reply:
Absolutely. Instead of changing that one place, I added a new eslint rule for similar cases and ran eslint --fix to quickyl fix all unnecessary usages of let throughout the whole code base. There were a bunch.